trigger/netdevices/__init__.py

```diff
@@ -565,31 +570,53 @@ def get_results(self):
             pass
         return self._results
 
-    def run_commands(self, commands):
+    def run_channeled_commands(self, commands, on_error=lambda x: x):
```

Comment:
Having a function in the method signature scares me. I feel like this should be `None` as a sentinel and then...

```python
 if on_error is None:
    on_error = lambda x: x
```